playground/mocks/idp/idx/data/factor-enroll-password.json

```diff
@@ -16,12 +16,11 @@
         "method": "POST",
         "value": [
           {
-            "name": "credential",
+            "name": "credentials",
             "form": {
               "value": [
                 {
-                  "name": "value",
-                  "label": "Password",
+                  "name": "passcode",
```

Comment:
I think this is breaking some of our e2e tests. https://travis-ci.org/okta/okta-signin-widget/jobs/605603470?utm_medium=notification&utm_source=github_status. You might have to update the selectors.

Reply:
yep, will fix